test-tools/pom.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.kie.cloud.tests</groupId>
```

Comment:
for a test suite we are using groupId org.kie.cloud - do we want to use same groupId structure? @sutaakar wdyt?

Reply:
Well, we can base the group id based on repo where it is placed, for example org.kie.cloud.tools , just a suggestion.

Reply:
will update it